backend/game/player/player.py

```diff
@@ -32,3 +33,21 @@ def update_energy_by(self, change_integer):
         self.energy += change_integer
         if self.energy < constants.DEFAULT_ENERGY_CUBE:
             self.energy = constants.DEFAULT_ENERGY_CUBE
+
+    def add_card(self, card):
+        self.card_on_hand[card.name] = card.effect
+
+    def remove_card(self, card):
+        self.card_on_hand.pop(card.name)
+
+    def get_card_at_index(self, index):
+        return self.card_on_hand.pop(index)
+
+    def get_current_hand(self):
+        return self.card_on_hand
+
+    def discard_all_cards(self):
+        self.card_on_hand.clear()
+
+    def lose_all_stars(self):
```

Comment:
This function is a bit unnecessary, what could be better is to have a reset to default.